app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __repr__(self) -> str:
+        return f'{{Name: {self.name}, ' \
+               f'Health: {self.health}, ' \
+               f'Hidden: {self.hidden}}}'
```

Comment:
You can use () instead of /, but I'm not sure if this is best practice

Reply:
I'll try to leave it as is for now (imho, more readable in scope of curly brackets symbol escaping)